docs/third-party-developers/extensibility/hooks/actions.md

```diff
@@ -315,6 +316,21 @@ do_action( 'woocommerce_blocks_enqueue_checkout_block_scripts_before' )
 
 ---
 
+## woocommerce_blocks_loaded
+
+Fires when the woocommerce blocks are loaded and ready to use.
+
+```php
+add_action( 'woocommerce_blocks_loaded', 'my_extension_woocommerce_blocks_support' );
+```
+```php
+do_action( 'woocommerce_blocks_loaded' )
+```
```

Comment:
I think the structure can be improved here:
- `Fires when the woocommerce blocks are loaded and ready to use.` is related to do_action, from the order here it looks like it's related to add_action
- we can use add action to provide an example, just like [here](https://github.com/woocommerce/woocommerce-blocks/blob/00a1174aebefae2b37178a3ded68cc6c8d56bbf4/docs/third-party-developers/extensibility/hooks/actions.md#example)


The expected result would be something like...

Fires when the woocommerce blocks are loaded and ready to use.

```php
do_action( 'woocommerce_blocks_loaded' )
```
### Example

# Do something when blocks is loaded

```php
// The action callback function.
function my_function_callback( ) {
// start doing your WC Blocks related logic
}

add_action( 'woocommerce_blocks_loaded', 'my_function_callback' );
```

Reply:
This change is not needed, the change from @senadir took care of my suggestion, and we can skip the example.